CI is broken (disabled private-lib in firejail.config)

Apparently caused by commit 9741d0b60 ("fix disabled private-lib in
/etc/firejail/firejail.config", 2022-06-23).

From its pipeline at <https://github.com/netblue30/firejail/runs/7030862406?check_suite_focus=true>:

```
TESTING: private-lib (test/fs/private-lib.exp)
spawn /bin/bash
firejail --private-lib --private-bin=sh,bash,dash,ps,grep,ls,find,echo,stty 
runner@fv-az489-993:~/work/firejail/firejail/test/fs$ 
<private-bin=sh,bash,dash,ps,grep,ls,find,echo,stty   
Error: private-lib feature is disabled in Firejail configuration file
runner@fv-az489-993:~/work/firejail/firejail/test/fs$ TESTING ERROR 1
[...]
make[1]: *** [Makefile:6: fs] Error 1
make: *** [Makefile:260: test-fs] Error 2
[...]
##[error]Process completed with exit code 2.
```

Relates to #5190.

Cc: @netblue30 @reinerh

--- a/test/fs/fs.sh
+++ b/test/fs/fs.sh
@@ -46,8 +46,12 @@ echo "TESTING: read/write /var/tmp (test/fs/fs_var_tmp.exp)"
 rm -f /var/tmp/_firejail_test_file
 
 if [ "$(uname -m)" = "x86_64" ]; then
+    fjconfig=/etc/firejail/firejail.config
+    printf 'private-lib yes\n' | sudo tee -a "$fjconfig" >/dev/null
     echo "TESTING: private-lib (test/fs/private-lib.exp)"
     ./private-lib.exp
+    printf '%s\n' "$(sed '/^private-lib yes$/d' "$fjconfig")" |
+      sudo tee "$fjconfig" >/dev/null
 else
     echo "TESTING SKIP: private-lib test implemented only for x86_64."
 fi

